.github/workflows/lint.yml

```diff
@@ -0,0 +1,27 @@
+---
+name: lint
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+concurrency:
+  group: lint-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  chefstyle:
```

Comment:
Let's rename the job as "lint" since chefstyle is no longer being used

Reply:
done